### PR TITLE
Add Slovak (sk) translation

### PR DESCRIPTION
**Pull request by jooray**
Adding Slovak (sk) translation for Android app

### CODE REVIEW

**Review by dpad85 — APPROVED** [review]
LGTM, thanks for the PR @jooray

File: phoenix-android/src/main/res/values-sk/strings.xml [hunk 1]
```diff
@@ -0,0 +1,773 @@
...
+
+    <string name="inappnotif_title">Notifikácie</string>
+    <string name="inappnotif_notices_title">Dôležité správy</string>
+    <string name="inappnotif_payments_title">Nedávna aktivita</string>
+    <string name="inappnotif_empty">Zatiaľ žiadne oznámenia</string>
+
+    <!-- settings: display seed -->
+
+    <string name="displayseed_title">Obnovovacia fráza</string>
+    <string name="displayseed_loading">Odomýkám seed…</string>
```
Comment (webiumsk) [inline phoenix-android/src/main/res/values-sk/strings.xml:293]: Odomykám - bez dĺžňa na y

File: phoenix-android/src/main/res/values-sk/strings.xml [hunk 2]
```diff
@@ -0,0 +1,773 @@
...
+    <string name="channeldetails_receivable">Likvidita na prichádzajúce platby</string>
+
+    <string name="channeldetails_commitments">Aktívne záväzky</string>
+    <string name="channeldetails_inactive_commitments">Neaktívne záväzky</string>
+    <string name="channeldetails_commitment_funding_tx_id">Financujúca tx:</string>
+    <string name="channeldetails_commitment_balance">Zostatok:</string>
+    <string name="channeldetails_commitment_funding_capacity">Kapacita:</string>
+    <string name="channeldetails_commitment_linked_payments">Vyvolané:</string>
+
+    <string name="channeldetails_json">Zobraziť holé dáta</string>
```
Comment (webiumsk) [inline phoenix-android/src/main/res/values-sk/strings.xml:249]: Be consistent: Zobraziť holé dáta vs. Importovať surové údaje kanála

Comment (jooray) [reply phoenix-android/src/main/res/values-sk/strings.xml:249]: Very cool. Make pull request with the changes if you can.

Comment (jooray) [reply phoenix-android/src/main/res/values-sk/strings.xml:249]: Creating pull request with these changes, thank you!

**Pull request merged**